install.sh

```diff
@@ -4,23 +4,25 @@
 
 set -e
 
-case $(uname -s) in
+case "$(uname -s)" in
 Darwin) target="x86_64-apple-darwin" ;;
 *) target="x86_64-unknown-linux-gnu" ;;
 esac
 
-if [ $(uname -m) != "x86_64" ]; then
+if [ "$(uname -m)" != "x86_64" ]; then
 	echo "Unsupported architecture $(uname -m). Only x64 binaries are available."
 	exit
 fi
 
-if [ $# -eq 0 ]; then
-	deno_asset_path=$(
+if [ "$#" -eq 0 ]; then
```

Comment:
This change doesn't prevent word splitting either. `$#` is guaranteed to be a positive natural number.

Please revert.

Reply:
I do not pick and choose. My strategy is to quote every variable.